doc/en/developer/source/cite-test-guide/index.rst

```diff
@@ -8,226 +8,72 @@ A step by step guide to the GeoServer Compliance Interoperability Test Engine (C
-     
-   With the following parameters:
-   
-   #. ``Capabilities URL``:
-
-         http://localhost:8080/geoserver/wcs?service=wcs&request=getcapabilities&version=1.1.1
-   
-   Click ``Next``
-
-   .. image:: tewcs-1_1a.jpg
-
-#. Accept the default values and click ``Submit``
-
-   .. image:: tewcs-1_1b.jpg
-
-Run WCS 1.0 tests
------------------
-
-.. warning:: 
-
-   The WCS specification does not allow a cite compliant WCS 1.0 and
-   1.1 version to co-exist. To successfully run the WCS 1.0 cite tests
-   the ``wms1_1-<VERSION>.jar`` must be removed from the geoserver 
-   ``WEB-INF/lib`` directory.
+   .. note::
+      Valid values for the ETS parameter are
+        * wcs10
+        * wcs11
+        * wfs10
+        * wfs11
+        * wms11
+        * wms13
    
-#. Remove the ``wcs1_1-<VERSION>.jar`` from ``WEB-INF/lib`` directory.
-
-#. Start GeoServer with the ``citewcs-1.0`` data directory.
-
-#. Change directory back to the cite tools and run the tests::
-
-     ant wcs-1.0
-     
-   With the following parameters:
-
-   #. ``Capabilities URL``:
-        
-          http://localhost:8080/geoserver/wcs?service=wcs&request=getcapabilities&version=1.0.0
-
-   #. ``MIME Header Setup``: "image/tiff"
-
-   #. ``Update Sequence Values``:
-
-      * "2" for ``value that is lexically higher``
-      * "0" for ``value that is lexically lower``
-
-   #. ``Grid Resolutions``:
+   Choose which GeoServer war to test by setting the GEOSERVER_WEBAPP_SRC environment variable::
 
-      * "0.1" for ``RESX``
-      * "0.1" for ``RESY``
+    GEOSERVER_WEBAPP_SRC=https://build.geoserver.org/geoserver/master/geoserver-master-latest-war.zip
 
-   #. ``Options``:
+#. Build the testing suite::
   
-      * Ensure ``Verify that the server supports XML encoding`` is *checked*
-      * Ensure ``Verify that the server supports range set axis`` is *checked*
-
-   #. ``Schemas``:
-
-      * Ensure that ``original schemas`` is selected
-
-   #. Click ``OK``
-
-   .. image:: tewcs-1_0a.jpg
-
-   .. image:: tewcs-1_0b.jpg
-
-.. _commandline:
+    # FROM THE GEOSERVER CODE BASE ROOT FOLDER
+    pushd build/cite
+    docker_command="docker-compose -f docker-compose.yml -f ./${ETS}/docker-compose.override.yml "
+    ### Cleaning up previous runs
+    eval $docker_command  down --rmi all -v
+    eval $docker_command  rm -vfs
+    
+    ### Build docker images
+    eval $docker_command  build --build-arg "GEOSERVER_WEBAPP_SRC=${GEOSERVER_WEBAPP_SRC}" geoserver
+
+#. Run the test::
+  
+    eval $docker_command  up --force-recreate teamengine
 
-Teamengine Web Application
---------------------------
+How to run TEAM Engine standalone
+---------------------------------
+To run a standalone version of TEAM Engine, start it with the following command::
 
-The Teamengine web application is useful for analyzing results of a test run. To run the web application execute::
+    docker run -d --name standalone_teamengine -p 8080:8080 geosolutionsit/teamengine:latest
```

Comment:
Is this a specific build of the team engine? I thought OGC offered a docker bundle of the team engine?

If they do not have a nightly build of the team engine we can offer to do so on build.geoserver.org and publish to osgeo docker repo.

Reply:
@jodygarnett it's a specific build that we are creating on our build server based on tagged sources of the team engine.
My experience dealing with the CITE testing team has been pretty poor, tickets ignored for months or more, and then they got closed within days if I failed to comment on them after a question from them. I strongly recommend we stay independent.